src/stories/components/CategoryChip/CategoryChip.tsx

```diff
@@ -70,6 +81,19 @@ const colors: { [id: string]: any } = {
 
 export const CategoryChip = (props: StatusChipProps) => {
   const { isLight } = useThemeContext();
+  if (!colors[props.category]) {
+    <Chip
+      style={{
+        color: isLight ? colors.Default.color : colors.Default.colorDark,
+        borderColor: isLight ? colors.Default.color : colors.Default.colorDark,
+        background: isLight ? colors.Default.background : colors.Default.backgroundDark,
+        ...props.style,
+      }}
+    >
+      {props.category}
+    </Chip>;
+  }
+
```

Comment:
Why not just create a variable or any other method to avoid duplicating the `Chip` code? Maybe something like this:

```tsx
const palette= colors[props.category] ? colors[props.category] : DEFAULT;

<Chip
      style={{
        color: isLight ? palette.color : palette.colorDark,
        borderColor: isLight ?palette.color : palette.colorDark,
        background: isLight ? palette.background : palette.backgroundDark,
        ...props.style,
      }}
    >
      {props.category}
    </Chip>;
```

Reply:
you are right, let me take this approach